src/components/css/BookItem.module.css

```diff
@@ -1,39 +1,166 @@
-.BookContainer {
-  width: 80%;
-  margin: 0 auto;
-  font-size: 1.75em;
-  border: 1px solid black;
-  display: flex;
-  padding: 1em;
-  justify-content: space-around;
-}
-
-.BookContainer .BookInfo {
-  display: flex;
-  flex-direction: column;
-  gap: 0.5rem;
-}
-
-.BookContainer .BookInfo ul {
-  list-style: none;
-  display: flex;
-  gap: 1em;
-  padding-top: 1em;
-}
-
-.BookContainer .BookInfo ul li a {
-  text-decoration: none;
-}
-
-.Oval {
-  width: 4.25rem;
-  height: 4.25rem;
-  border-style: solid;
-  border-radius: 50%;
-  border-width: 5px;
-  border: 5px solid #307bbe;
-  border-left-color: #e8e8e8;
-  border-image-source: linear-gradient(to bottom, #307bbe, var(--dodger-blue));
-  border-image-slice: 1;
-  transform: rotate(45deg);
-}
+.BookContainer {
+  width: 80%;
+  height: 8rem;
+  margin: 0 auto;
+  font-size: 1.75em;
+  border: 1px solid #e8e8e8;
+  border-radius: 4px;
+  display: flex;
+  padding: 1em;
+  justify-content: space-around;
+  background-color: #fff;
+  font-family: Montserrat, sans-serif;
+}
+
+.BookContainer .BookInfo {
+  display: flex;
+  flex-direction: column;

```

Comment:
- [ ] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code to No repeating of the same rule by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```css
.d-flex {
  display: flex;
}

.flex-center {
  align-items: center;
  justify-content: center;
}
```
- JS file
```js
<div className="d-flex flex-center">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉

Reply:
I was a bit confused how to add multiple classes using modular CSS but now I think I have and idea.